release-info/News

```diff
@@ -1,3 +1,44 @@
+OCaml 5.4.0 (? October 2025)
+----------------------------
+
+Some of the highlights in OCaml 5.4.0 are:
+
+- labelled tuples
+- immutable arrays
+- array literal syntax for immutable arrays and `floatarray`s
```

Comment:
```suggestion
- array literal syntax for immutable arrays and `floatarray`s using type disambiguation
```
otherwise it feels like you're saying that there are new dedicated syntax for them